packages/api/src/web3Middleware/provider.ts

```diff
@@ -16,6 +16,7 @@ export function getProvider (network: Network) {
     [Network.HarmonyTestnet]: process.env.HARMONY_TESTNET_PROVIDER,
     [Network.KCCTestnet]: process.env.KCC_TESTNET_PROVIDER,
     [Network.KCCMainnet]: process.env.KCC_MAINNET_PROVIDER,
+    [Network.MoonbeamMoonbase]: process.env.MOONBASE_ALPHA_PROVIDER,
```

Comment:
same as before, as a general comment to be considered as input for upcoming issue:
- why not reading the provider url from config file ? this way no code is to be added every time a new network is added to config